benchmarks/dali_vs_python/BM_rn50/run_benchmark.py

```diff
@@ -0,0 +1,126 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import tritonclient.grpc as tritonclient
+import argparse
+import sys
+import re
+import subprocess
+
+
+def create_cmd(cmd_options):
+    cmd = ["perf_analyzer"]
+    for option, value in cmd_options.items():
+        cmd.extend([option, value])
+    return cmd
+
+
+def check_sample_shape(input_name):
+    input_data_dir = 'test_sample'
+    ret = subprocess.run(['stat', '--printf', '%s', f'{input_data_dir}/{input_name}'],
+                         capture_output=True, check=True)
+    return int(ret.stdout)
+
+
+def create_cmd_options(model_name, input_name, batch_size, sample_shape):
+    input_data_dir = 'test_sample'
+    cmd_options = {
+        '-m': model_name,
+        '-b': batch_size,
+        '--input-data': input_data_dir,
+        '--shape': f'{input_name}:{sample_shape}',
+    }
+    return cmd_options
+
+
+def parse_perf_analyzer_result(pa_result):
+    def search(regex):
+        return re.search(pattern=regex, string=pa_result).group()
+
+    ret = {
+        'batch_size': search(r'(?<=Batch size: )\d+'),
+        'concurrency': search(r'(?<=Concurrency: )\d+'),
+        'overhead': search(r'(?<=overhead )\d+'),
+        'queue': search(r'(?<=queue )\d+'),
+        'compute_input': search(r'(?<=compute input )\d+'),
+        'compute_infer': search(r'(?<=compute infer )\d+'),
+        'compute_output': search(r'(?<=compute output )\d+'),
+        'server_latency': search(r'(?<=Avg request latency: )\d+'),
+        'client_latency': search(r'(?<=Avg latency: )\d+'),
+        'client_throughput': search(r'(?<=Throughput: )\d+')
+    }
+    return ret
+
+
+def generate_csv(result_dicts):
```

Comment:
`perf_analyzer -f file.csv` outputs the resuts to the csv

Reply:
That's true, however single `perf_analyzer` run is defined per single batch size. Therefore, I'd still need to call `perf_analyzer` multiple (say 10) times, create 10 CSVs (one per every batch size), open them, parse and write all the data to a new CSV. Judging from this, parsing the console output seems easier. Unless you'd have a better idea? I'm open to any suggestions

Reply:
Ok, seems reasonable